pkgs/development/tools/build-managers/bazel/bazel-deps/default.nix

```diff
@@ -0,0 +1,57 @@
+{ stdenv, buildBazelPackage, lib, fetchFromGitHub, git, jre, makeWrapper }:
+
+buildBazelPackage rec {
+  name = "bazel-deps";
+  version = "2018-05-31";
+
+  src = fetchFromGitHub {
+    owner = "johnynek";
+    repo = "bazel-deps";
+    rev = "dd7d0086d3a61c1d5c3370a0300824d0c75946e4";
+    sha256 = "1h9ddgk6vn0bhnnkwwz4n1iqv2rcdj521dxhdwj5wwpndbciw855";
+  };
+
+  bazelTarget = "//src/scala/com/github/johnynek/bazel_deps:parseproject_deploy.jar";
+
+  buildInputs = [ git makeWrapper ];
+
+  fetchAttrs = {
+    preInstall = ''
+      rm -rf $bazelOut/external/{bazel_tools,\@bazel_tools.marker,local_*,\@local_*}
+      pushd $bazelOut/external
+        files_to_delete=()
+        for workspace in $(find . -name "WORKSPACE" -maxdepth 2 -print0 | xargs -0L1 dirname); do
+          pushd "$workspace"
+            mapfile -t referenced_files < <(comm -23 <(bazel query '//...:*' | sort -u) <(bazel query '//...:all' | sort -u) | sed -e 's,^//,./,g' | sed -e 's,:,/,g') || continue
+            for referenced_file in "''${referenced_files[@]}"; do
+              if target=$(readlink "$referenced_file"); then
+                rm "$referenced_file"
+                cp -a "$target" "$referenced_file"
+              fi
+            done
+            mapfile -t workspace_files_to_delete < <(find . -type f -or -type l | sort -u | comm -23 - <(printf '%s\n' "''${referenced_files[@]}" | sort -u))
+            for workspace_file_to_delete in "''${workspace_files_to_delete[@]}"; do
+              files_to_delete+=("$workspace/''$workspace_file_to_delete")
+            done
+            bazel shutdown
+          popd
+        done
+        for file_to_delete in "''${files_to_delete[@]}"; do
+          rm "$file_to_delete"
+        done
+        find . -type d -empty -delete
+      popd
+    '';
+
+    sha256 = if stdenv.isDarwin then "0d33zh40wk2qm7z7a777470k8dqiki0dlj00mqrl475kydc0xfxc" else "080lz22irpqryrx09m66kv35aqz2fk1xsz4yhwz56h9rpny8d9ac";
+  };
+
+  buildAttrs = {
+    installPhase = ''
+      mkdir -p $out/bin/bazel-bin/src/scala/com/github/johnynek/bazel_deps
+      cp gen_maven_deps.sh $out/bin
+      wrapProgram "$out/bin/gen_maven_deps.sh" --set JAVA_HOME "${jre}" --prefix PATH : ${lib.makeBinPath [ jre ]}
+      cp bazel-bin/src/scala/com/github/johnynek/bazel_deps/parseproject_deploy.jar $out/bin/bazel-bin/src/scala/com/github/johnynek/bazel_deps
+    '';
+  };
+}
```

Comment:
I'm not familiar with buildBazelPackage, but it seems like the meta attribute might be missing here.